src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -193,15 +197,39 @@ export class NbChatMessageComponent {
    */
   @Input() type: string;
 
+  /**
+   * Data will be set as custom message template context
+   * @type {any}
+   */
+  @Input() customMessageData: any;
+
+  protected readonly defaultMessageTypes: string[] = ['text', 'file', 'map', 'quote'];
+
   constructor(protected domSanitizer: DomSanitizer) { }
 
   getInitials(): string {
     if (this.sender) {
       const names = this.sender.split(' ');
-
       return names.map(n => n.charAt(0)).splice(0, 2).join('').toUpperCase();
     }
-
     return '';
   }
+
+  isDefaultMessageType(type: string): boolean {
+    return type === null || type === undefined || this.defaultMessageTypes.includes(type);
```

Comment:
```suggestion
    return type == null || this.defaultMessageTypes.includes(type);
```
We usually check for `null` and `undefined` with a single comparison like above.